common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -1022,176 +1024,177 @@ class StudentAdminPage(PageObject):
     Student admin section of the Instructor dashboard.
```

Comment:
Note to reviewers: I refactored the `StudentAdminPage` since it was confusing with buttons for different sections of the page - and mostly only for the Entrance exam portion.

I have instead created base implementations of all the buttons and input boxes and created the following subclasses for each of the sections: `StudentSpecificAdmin`, `CourseSpecificAdmin`, `EntranceExamAdmin`.